src/System.Private.ServiceModel/tools/test/Bridge/Bridge/ConfigController.cs

```diff
@@ -16,14 +16,25 @@ namespace Bridge
 {
     public class ConfigController : ApiController
     {
-        private const int Default_BridgeIdleTimeoutMinutes = 20;
+        public static EventHandler<Tuple<BridgeConfiguration, BridgeConfiguration>> BridgeConfigurationChanged;
```

Comment:
I liked @SajayAntony 's suggestion of a ChangedEventArgs<T> because it allowed a better factoring of his 2nd suggestion, to remove knowledge of BridgeConfiguration from the IdleTimeoutManager.  Now, the Bridge has separate events for changed resource folder and changed idle timespan, and clients sign up for only the changes they care about.